spec/rspec/core/did_you_mean_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'tmpdir'
+require 'rspec/support/spec/in_sub_process'
+
+module RSpec
```

Comment:
We favour `RSpec::Core` style name spacing, saves some spaces :)

Reply:
I prefer it too, but when I used it, I encountered the following cop:
`Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.`
When I investigated further, I found this [description](https://github.com/houndci/hound/issues/292) of why the cop is there.

Reply:
Weird, we do that style elsewhere, is it possible you're using more updated version of Rubocop than us? Or not running with our config?

Reply:
I am using `0.52.1`, the same as in the Gemfile. I looked at .rubocop.yml and it has the following entry for this cop:
```
# Not sure what to do with this rule yet.
Style/ClassAndModuleChildren:
  Exclude:
    - lib/rspec/core/formatters.rb
    - lib/rspec/core/notifications.rb
    - lib/rspec/core/option_parser.rb
    - lib/rspec/core/reporter.rb
```
so I can either add `did_you_mean.rb` to this list, or leave the nested modules syntax in?

Reply:
No its not a blocker